src/commands/config/prompts.js

```diff
@@ -41,5 +42,11 @@ export default () => [
     message: 'Enable scope prompt',
     type: 'confirm',
     default: configurationVault.getScopePrompt()
+  },
+  {
+    name: CONFIGURATION_PROMPT_NAMES.SKIP_PROMPTING_FILLED_INFO,
+    message: 'Skip prompting already filled info',
```

Comment:
I think we need to clarify better what this option is doing. For you this might be already familiar but for someone who uses the cli and doesn't looks into the PR issues I don't think `Skip prompting already filled info` is an understandable message, what information is referring this to ?

Reply:
Sure perhaps a more verbose sentence of what it does ?

> Skip prompting for gitmoji, title and message commit when retrieved